changes-set.txt

```diff
@@ -90,6 +90,46 @@ make a github issue.)
 DONE:
 Date      Old       New         Notes
  1-Apr-25 cxpgt0d   [same]      Moved from SN's mathbox to main set.mm
+30-Mar-25 df-rnghom df-ringhm
+30-Mar-25 df-rngiso df-ringim
```

Comment:
I think you also need to update `change-set.txt` with the new labels.

Reply:
You are right, of course.

Reply:
Fixed